app/controllers/course/stage/code-examples.ts

```diff
@@ -95,6 +97,13 @@ export default class CodeExamplesController extends Controller {
       return;
     }
 
+    // TODO: Remove: Temporary measure for private course
```

Comment:
@ryan-gang let's do this on the backend instead - add a sentry event so that we know the route was accessed. It'd be useful to know the exact details of how many people accessed this and who they were, if the number is high we might have a link somewhere that leads to this route.